### PR TITLE
Awesome Chrome Extension Website

### PR DESCRIPTION
**Pull request by praniti111**
### 👨‍💻 Changes proposed

1. Created a website for displaying all the extensions in one place.
2. Search Bar Added.
3. Dark and Light mode available.

### Tech Stack Used:
HTML, CSS, JavaScript

### ✔️ Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots
Awesome Chrome Extentions - Google Chrome 25-03-2022 15_38_30

### CODE REVIEW

File: Awesome Chrome Extensions Website/index.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+  <link rel="stylesheet" href="style.css" />
+  <link rel="shortcut icon" href="favicon.png" type="image/x-icon">
+
+  <script language="JavaScript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.10.0/jquery.min.js"></script>
+
+  
+  <script src="https://kit.fontawesome.com/a076d05399.js"></script>
+  <!--<script src="script.js"></script>-->
+  
+  <title>Awesome Chrome Extentions</title>
```
Comment (ridsuteri) [inline Awesome Chrome Extensions Website/index.html:17]: Typo : Extensions

File: Awesome Chrome Extensions Website/index.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+</head>
+<body id="body" class="dark-mode">
+    <!--<h2 class="shadow"><a href="index.html">Awesome Chrome Extentions</a></h2>-->
+    <nav id="body" class="dark-mode">
+        <input type="checkbox" id="check">
+        <label for="check" class="checkbtn">
+          <i class="fas fa-bars"></i>
+        </label>
+        <label class="shadow">Awesome Chrome Extentions</label>
```
Comment (ridsuteri) [inline Awesome Chrome Extensions Website/index.html:26]: Typo: Extensions

File: Awesome Chrome Extensions Website/index.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+       ></path></svg></a
+   >
+ <!-- Octocat svg end -->
+ 
+ <!-- Scroll To Top -->
+ <a class="scrollToTop"><span></span></a>
+ <!-- Scroll to Top End -->
+
+ <!-- Heading -->
+ <h1>Find the most useful and creative Chrome Extentions</h1>
```
Comment (ridsuteri) [inline Awesome Chrome Extensions Website/index.html:80]: Typo: Extensions

**Review by ridsuteri — COMMENTED** [review]
looks good, just correct some minor typos :)

**Comment by praniti111** [comment]
@ridsuteri done👍

**Review by ridsuteri — APPROVED** [review]

**Pull request merged**